src/clr-angular/forms-deprecated/forms.module.ts

```diff
@@ -7,9 +7,9 @@
 import {CommonModule} from "@angular/common";
 import {NgModule} from "@angular/core";
 
-import {ClrCommonFormsModule} from "../forms/common/common.module";
+import {ClrDatepickerModule} from "../forms/datepicker/datepicker.module";
 
 import {ClrCheckboxModule} from "./checkbox/checkbox.module";
 
-@NgModule({imports: [CommonModule], exports: [ClrCommonFormsModule, ClrCheckboxModule]})
+@NgModule({imports: [CommonModule], exports: [ClrCheckboxModule, ClrDatepickerModule]})
```

Comment:
You just dropped the `ClrCommonFormsModule` there, you need to put it back.

Reply:
Its imported from ClrDatepickerModule. I checked your Radios and Checkboxes PR and you did the same too.

Reply:
I can add it back but it will conflict with your PR.

Reply:
What I'm saying if `ClrFormsNextModule` should export it, but `ClrFormsModule` shouldn't.

Reply:
Cool. Will make those updates

Reply:
Fixed